samples/expo/package.json

```diff
@@ -18,23 +18,23 @@
   },
   "dependencies": {
     "@sentry/react-native": "6.13.0",
-    "@types/react": "~18.3.12",
-    "expo": "~52.0.42",
-    "expo-constants": "~17.0.8",
-    "expo-image-picker": "~16.0.5",
-    "expo-linking": "~7.0.5",
-    "expo-router": "~4.0.19",
-    "expo-status-bar": "~2.0.1",
-    "expo-updates": "~0.27.4",
-    "expo-web-browser": "^14.0.2",
-    "react": "18.3.1",
-    "react-dom": "18.3.1",
-    "react-native": "0.76.9",
-    "react-native-gesture-handler": "~2.20.2",
-    "react-native-safe-area-context": "4.12.0",
-    "react-native-screens": "~4.4.0",
-    "react-native-web": "~0.19.13",
-    "typescript": "^5.3.2"
+    "@types/react": "~19.0.10",
+    "expo": "^53.0.0",
+    "expo-constants": "~17.1.5",
+    "expo-image-picker": "~16.1.4",
+    "expo-linking": "~7.1.4",
+    "expo-router": "~5.0.5",
+    "expo-status-bar": "~2.2.3",
+    "expo-updates": "~0.28.12",
+    "expo-web-browser": "~14.1.6",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
+    "react-native": "0.79.2",
+    "react-native-gesture-handler": "~2.24.0",
+    "react-native-safe-area-context": "5.4.0",
```

Comment:
Just noting that I faced [this issue](https://github.com/AppAndFlow/react-native-safe-area-context/issues/618) with the latest `react-native-safe-area-context` version locally and was solved by bumping cocoapods to the latest version.